recipes/dust/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "dust" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: dust
+  version: {{ version }}
+
+source:
+  url: https://github.com/bootandy/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4d13a2edf38ab7bfc01b700f5856c92aa3772249a203b34247c3b2c7dd8fa574
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [unix]
+
+test:
+  commands:
+    - dust --help
+
+about:
+  home: https://github.com/bootandy/dust
+  license: Apache-2.0
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
As rust produces static compiled binaries, it also includes the code of various dependencies into the binary and thus requires their licenses to be included ... see other recently merged rust based recipes on how to do that.